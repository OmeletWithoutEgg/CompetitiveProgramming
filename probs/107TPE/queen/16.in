522
-100314764 -25767286
30483794 -42498288
125798198 -44550206
38597475 -27597373
-21924262 -87543946
-22928512 -88943236
125192070 -42626567
-97191334 -95615832
-13918632 -68418371
-65441922 -9152292
-88891001 -16295000
-40142318 -103542613
-16213132 -41570588
29405613 -71118385
-80309551 -106018185
-66848912 -9239810
35082717 -84570379
124876213 -75504201
-31761908 -20087216
125952409 -45087531
126085646 -45571053
47681171 -18713383
-56706925 -108691737
-39376740 -15049515
85691822 -9235909
-101506174 -91039569
67905189 -107619635
-76039009 -107375779
36622511 -86950434
-51342236 -107706622
-83560493 -13498856
-79585285 -106278404
-18258741 -36857431
31511724 -77476968
34749968 -33208999
114373428 -92722880
28950351 -48022638
-14704011 -46308895
109016439 -97682779
28125615 -64165714
45472107 -20482241
-17839541 -80466868
83136690 -108327876
-47041529 -106465490
-71688882 -9848072
-35473733 -100808090
41947667 -23777166
127568780 -63579669
-34518733 -18028298
-108432630 -80062628
-43401454 -13113520
127293882 -51403546
35216730 -84789721
-38191596 -15707132
125444513 -43395215
121334172 -33990923
-40995991 -103972146
-15794848 -42731807
-112995326 -61245790
-18564267 -36252471
110801383 -96188045
118707511 -87385013
40455649 -25382409
88715910 -107409260
75985968 -108578250
114411331 -24541304
-90852360 -17542715
-45723257 -12189695
36433017 -86672989
-39813682 -14817530
-41407299 -14017421
-37628909 -16034291
-100394895 -25857121
-15474769 -43687030
76959077 -8619106
82367099 -108404220
-38750626 -102797591
-105417484 -32556252
57567222 -104328048
-63743168 -9099566
-14639758 -46554327
113302176 -93835970
-65577845 -109030279
-79890145 -106170405
33509194 -81782640
106799209 -99354670
-86611358 -103190761
71828395 -8971591
127794070 -60098990
-16290118 -41366222
-72136816 -9928829
60739889 -105605373
-66733818 -108958135
41694310 -93183561
-42354364 -13575007
95202315 -105491634
76074688 -8642096
-42123006 -104508426
-75849012 -107426526
-70550158 -9661641
28930131 -48116384
-108996002 -78797306
73273826 -108405000
48905488 -99406012
27820555 -59272145
112128848 -94979901
82474370 -108394300
-88854643 -16273062
68780576 -9434959
126706146 -48134519
-64422627 -9113720
95388540 -11801368
126112479 -45670785
-43134720 -13228369
53740711 -102433798
-49906650 -10850759
83846254 -8976707
88203072 -107520988
-112265788 -50320740
81182788 -108498290
66771045 -9846969
92740557 -10891078
-112290444 -67729081
120261073 -32183891
110558344 -20823512
120369608 -32359008
127364529 -65317163
127565094 -53607073
-66329458 -9202860
-61211950 -109061156
28673470 -67830907
68934232 -9406974
-80787458 -12349749
117831584 -88591227
60224621 -105414927
30518997 -74828269
-47745398 -106697434
-103715685 -88174200
35946082 -85941116
93056360 -106232524
93585756 -11163672
107361726 -98948609
-60167582 -109011975
28222812 -64975444
-19867514 -84313615
-107715054 -36638367
32297010 -37923503
-60351711 -9167053
-14418738 -47440665
-28979968 -95697974
-56610467 -108679322
-19050282 -35329921
102497696 -15128159
27987702 -54473952
111878616 -22009144
65378515 -10183418
97411450 -12611477
117451528 -28131597
-39541556 -103227930
-14425514 -70776857
33486112 -35485067
-18194597 -81202210
-17109995 -39338995
-42887627 -13336405
36322042 -86508629
-113015364 -57474443
-30363011 -96937852
61904581 -11211120
118409847 -87803458
37429583 -29134280
-103522600 -88442394
-25297115 -26302080
122696182 -80652325
-100001386 -25420087
-113041587 -59036060
125643692 -44033337
36058090 -86111954
-79712465 -106233620
-23869111 -90167372
-95982051 -96710175
28568552 -49970588
105239149 -16802889
-46587379 -11879627
-84927696 -104050175
-83090702 -13290340
46069628 -19983329
27861135 -56492286
-101097289 -26663700
-72075037 -9917440
127787095 -56906176
29831099 -72662761
35611737 -31801615
109985458 -96888885
36578081 -86885741
27881178 -56063376
-112949134 -62134417
-19070567 -82896878
-16887291 -39865038
98429790 -104164801
127452756 -52594144
37145041 -29528132
-111068316 -73002793
-27092844 -24342877
43635620 -95104065
-76729724 -11004772
-37333608 -16209938
-111381543 -46317635
96121933 -105140243
32491913 -37499922
113013779 -23099366
-19792520 -34004503
-31345653 -97764602
-20434416 -85260005
-112679729 -53089789
100134600 -13869288
-78578274 -11569783
114615618 -24762056
127774855 -56579004
-18749232 -35895846
-80018322 -106124333
121141353 -83570336
123933263 -39292963
-56396130 -108651051
-64734426 -109065980
-31163410 -97614504
-18319129 -36736230
-77261761 -11159400
80240686 -108552942
-51904927 -107838609
122993518 -37187255
-110212608 -75674694
45130842 -20774372
-48253423 -106857697
55616635 -13810200
56188988 -103692397
-33674865 -18627452
-111301492 -72170765
-21098023 -31878206
122494237 -36164655
35333790 -32244251
100050398 -13827384
-74724604 -10478722
74638082 -8712865
-64196029 -9107973
-76347433 -107291692
-58526291 -9298944
99139308 -103837026
40223573 -25645186
73518994 -108426757
121492887 -34273590
-110977182 -44875571
-64680625 -109067774
86628021 -9394366
-101137700 -26711180
113955314 -93165542
98846312 -13249504
-29941510 -96569737
127318298 -65650341
-104880989 -86471045
-71941730 -9893140
36028102 -31157152
-55217096 -108478618
126779274 -68741749
39532556 -90773275
-47392104 -106582460
-69343216 -108695954
91524213 -106695957
44888227 -20985318
-38968633 -102918054
46770791 -19417588
-113004761 -61014202
113863839 -23962555
91616573 -10553986
108053068 -18790515
124584840 -76294906
85460332 -9199542
-15771248 -42800113
-90533244 -100858393
-107705855 -36620075
115548776 -91418042
-61767452 -9110882
-100898925 -91756925
-19349041 -83404280
-110536155 -43465527
-20292957 -85028247
94854472 -105619173
109408241 -19856855
-32904608 -19197772
-90096932 -17046931
83329219 -108306895
73543814 -8794625
95334799 -105442288
-34656876 -100256596
109674893 -20075759
51697357 -15976018
-97552223 -95275004
49915713 -17120034
-13787005 -50493324
-109932787 -41738995
-33992757 -99790633
121378345 -34069164
-55174646 -9717416
78196397 -108610313
-17062298 -78739029
-29779192 -21763548
57461059 -104280878
126103877 -45638725
-34311271 -100016117
97465397 -12634494
-63884679 -9101752
-32347102 -19625081
-15799900 -42717228
71881115 -8965260
50217507 -16918683
32039729 -38499193
-13160154 -62535454
-25928373 -92599969
121450168 -34197086
-15156686 -44706005
-62541425 -9097147
27954807 -62332260
-67273480 -108915236
-43393333 -13116989
29956356 -73083662
-109304563 -40127826
-112705876 -53310027
-73375129 -10174111
-111462963 -71559542
43815787 -95271987
-41909535 -13779803
-61298946 -109064266
-20717221 -85715034
-13136878 -56009743
30928092 -75975684
121134705 -83581872
-71772155 -108326522
-99197768 -93630612
76348253 -8633312
-33324087 -99304943
-76705658 -10997928
85237824 -9165632
28195298 -52466291
42398705 -23318678
-13072766 -60859148
30865464 -41417892
124432319 -76693153
126208158 -71190223
94877090 -105610969
46916647 -97921020
-16973001 -78528692
46654419 -19510047
-35555750 -17327111
41954200 -93453077
-88728298 -16197154
-67805382 -9322096
41749606 -93241244
93113336 -11009265
28826098 -68610285
89530721 -10003428
-108906035 -39183452
67905963 -107619792
-38289547 -15651177
-112237384 -68026437
-48638558 -11214045
-112747743 -64507725
-14374827 -70563867
27982701 -54534618
-52533664 -107978004
123370165 -79223301
37221451 -87801942
-82540032 -13053229
-101829924 -27543613
-72223961 -108244259
-109300927 -40118958
-17128139 -39296864
-30145286 -21440547
51923276 -15838440
-24228571 -27574062
74160280 -8745595
29121986 -69964036
126292114 -46360833
-19161566 -83064224
-108326093 -80291735
37337515 -87962679
-62744561 -9095527
38864862 -27262216
120160812 -85199976
29306028 -70726371
76398406 -108591653
74072089 -108471378
119637568 -31207878
-98201268 -94644599
-13551889 -51969590
-61547354 -109072311
-111052423 -45131730
89533755 -107219358
-18839820 -35723706
-95380278 -20960522
-86186042 -103415472
30573116 -42238255
105038253 -100551709
33288551 -81355748
-14335284 -47794526
-69988584 -9579599
-112188094 -68293790
29129271 -69995237
-18416309 -36542893
-65301190 -109043562
40492534 -25340984
-50305306 -10743985
-80132898 -106082816
-72368426 -108217050
-64503414 -109073272
-112451399 -66754531
-111741211 -70423811
119411085 -86358200
-63987521 -9103593
122841661 -36869926
104505889 -16331028
126992990 -49576123
127576537 -63501360
28544990 -67117549
126930929 -67978870
-104131465 -30605616
-78664916 -106591094
125269496 -42857900
36645607 -86983978
61161327 -11467136
-85688421 -14517493
-107203202 -82541495
104948146 -16613459
38421696 -89402561
76244204 -8636476
-57173235 -108749071
-35781126 -101009622
-87149976 -102898608
-111170496 -72644976
104767820 -100726006
-110038853 -42028297
88759956 -9824116
-110582597 -74581917
112468918 -94656018
94168220 -105862274
-57728983 -108811602
-109904306 -41662238
-106714092 -34748899
111139932 -21335379
98328445 -13012992
27884482 -55999447
123484029 -78969773
105177301 -16762388
-13048607 -59930382
126740791 -48297507
118735271 -87345521
97439666 -104600010
-13064897 -57569177
96165181 -105123205
-111595080 -71034562
-103758929 -30075670
-78185994 -106745961
-104426670 -31036174
44301826 -21506676
68566574 -9474761
-112701530 -64916456
49406362 -17467118
106529204 -17678117
-69910103 -108620637
95255860 -11751775
97034339 -12452654
27855397 -56632168
-24705181 -91193202
-13762044 -67551787
-14360182 -47687742
-13389794 -64984979
93226115 -11045654
86904383 -9444648
-14828384 -45847618
126008811 -45289912
101256155 -14446521
-25893730 -92561556
92642697 -10860583
-13359246 -53467784
111106093 -95918361
127814306 -58177365
-32005141 -19893409
125718246 -72943311
56467996 -103825191
71163423 -108167189
104568941 -100852621
-113013161 -57407860
-112354349 -67356297
-70213469 -9611672
-45379485 -12318059
-22619605 -29665737
66782712 -9844328
34372139 -33860716
-95960982 -96728615
-53003397 -10112668
31706869 -39274484
-52144675 -10296524
-27990277 -94751386
102477220 -15116543
127110537 -66982406
-110550678 -74679557
-59016682 -9256909
54612216 -102901435
-84317052 -104342363
31298243 -40279266
123708368 -78458881
110294269 -96627211
29665607 -45138785
-112989555 -61375862
127364740 -51907910
64953549 -106928964
-79611475 -106269213
-44884602 -105681366
36760048 -30074189
-113040278 -59461121
76276288 -108588040
-110883984 -44565077
58637003 -104787062
35136495 -32564814
