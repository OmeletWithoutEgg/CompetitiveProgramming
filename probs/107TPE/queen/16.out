7384656.75 -58851913.65
