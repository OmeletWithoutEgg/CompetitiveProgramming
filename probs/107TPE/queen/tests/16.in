362
-46426490 10628929
-83746605 50038257
-44008593 55052522
-43262025 17680198
97162534 -100975833
93796977 -110728459
-60864965 -919121
-68494482 70733282
-40457509 33342233
-47439038 8971888
-64064561 -1051158
54749660 -77874177
-40483466 32692750
-52664419 2974240
97262037 -91927953
-71634430 1643448
-40425317 35900233
-75011278 66287310
95236852 -84467386
66549064 -69636535
60012459 -119364907
51529540 -83024460
-52115461 3446519
-80916575 13022901
71089639 -123498087
96954804 -102129864
65989636 -69817599
51541103 -109234840
-61041292 -945478
-64992656 71672162
-78442099 8663021
66191931 -122485152
90151420 -115789691
-54071903 1900373
92210023 -78997059
-45230475 57825300
-40528704 38882126
-40464866 33138701
-82987015 18165912
55100327 -77440809
59365382 -118891905
-82761896 17504434
-81308661 13862247
-82776107 17545248
-47618812 62020616
48673835 -93573037
-55823863 69902071
-53911996 2012868
55758778 -76674603
-84907952 26027321
-83806103 20901099
90311173 -76621959
49840733 -104856104
92773349 -79821353
-40491439 32523062
96764170 -89221128
-45859265 59062806
61924824 -120584171
84812991 -120216133
-54625564 69190580
96668105 -103417110
-68811397 116968
-65091839 -938480
-57072988 203744
49372294 -89137954
-50737238 4776779
-41010992 43617374
-65596327 71574821
-81559494 14426848
66251829 -122504684
-45777368 58907779
74279092 -68682538
-84323027 23005731
-40436731 34082078
-63112378 -1088217
51197242 -83736547
-45638556 12078547
97239152 -100470531
51445259 -109035076
69547120 -123303177
82999673 -121222072
66951596 -122719239
-79787092 10855464
-84928330 26152183
50768283 -84739412
53844486 -79085530
70932108 -68752148
96457857 -88011175
-49622880 6018880
58876042 -118511963
-40649225 30215843
58527298 -118228889
48602756 -94665889
-84219187 48168690
-65077450 -940581
75890681 -123402707
59638963 -119095874
-40423430 35625311
52806962 -111563738
-62994071 -1088308
62508938 -71327735
95648337 -106726846
62575375 -120943106
-85220466 28193442
-44451136 56121209
-59811163 -714583
-74214068 3667269
-42247680 21017380
-48249071 7787115
-60650348 -884011
-40619955 30558315
-63146350 -1088007
-52647232 67730942
-54049069 1916292
-62648333 71802357
-84929610 26160082
-58568629 -368385
-46866987 60837541
61417535 -120285299
66728958 -122653682
-85653131 34339764
90682959 -77044935
52251298 -110604815
-82433769 16594529
49436609 -103366490
-42870258 18869553
97023588 -90460470
-83892318 49492589
92458869 -112882158
58705790 -118375079
-85495256 39777056
92442960 -112905294
88068148 -74412250
-62499180 -1077879
-42193991 21221204
60238488 -72712835
-53070202 2644887
-42186519 49469653
96416902 -104371493
55387711 -77099070
55965531 -76445888
-85555915 38885437
-53820488 2078313
-52001958 3548111
95674316 -85578427
62170727 -71512403
68888145 -69049620
52368055 -110813693
97519925 -94789271
77257379 -69055147
-47938284 62491363
91813199 -113782385
93059241 -111969710
-73062761 2683106
-79984344 11210067
96847293 -89591147
-60695906 71611239
95844118 -106186337
81184506 -122030800
-72692253 2394830
75701753 -68808863
-76928442 64131480
48904066 -100649249
-52704484 67778511
-74198348 3652895
-40835609 42291082
96989297 -90280704
-55189123 1178869
60315776 -72659794
50694739 -84922445
77033239 -69012597
-76769331 64329045
68905680 -69046228
-85618470 33099887
-82567511 16958132
-53694180 68549564
48573058 -95589384
94244562 -109896896
48647017 -93920221
97450306 -93659320
65679400 -69925027
-40898278 27926039
72628823 -123582777
88765269 -117193448
48688974 -93396687
-66043920 -766077
85033119 -120079396
76568525 -123303415
54718595 -114321933
96990737 -90288136
68279750 -69176449
-83032392 18303283
-71172544 69372147
-62120366 71777594
72040030 -123563240
97547970 -96311468
-82981273 18148628
52195071 -81732691
61115684 -120099289
97307544 -92273227
79591115 -69643684
52748467 -111466805
-45902798 11574996
-56542798 448725
-52202839 67350255
-77037699 63993699
80081230 -69802018
83326101 -121056228
96085071 -105471298
-40564052 31293239
97292227 -92153581
-77471365 7290236
62207402 -120743380
-53447653 68366391
-82166725 54823536
-56509496 70254629
94276201 -82399897
-42409821 20421738
81240590 -122008588
-70791804 1117357
-79856348 59740591
48637791 -94053216
81519402 -70339809
-46975805 9703802
90836774 -115010019
48837999 -92053958
-74136026 67123327
70981712 -123488054
-78435644 62066140
-85056923 26985888
48689959 -93385615
-62633705 -1082441
-77458327 7272847
52108947 -110344370
-85626593 33321480
-80036004 11304494
58571140 -118265054
84155875 -71630437
-61098543 -953553
-84281300 22820669
-73947900 3427467
-80085684 11395926
49113573 -90354628
-83462271 51038365
51358584 -83384468
48667103 -93655549
-61311189 71700978
92529028 -112779514
-70310259 69875218
63681792 -70741625
-41164376 44615857
-85390769 40987091
-84181623 22390698
52503403 -81184671
-61524938 71725776
-49985011 65121834
-40795471 41948416
88259482 -74580421
-77696605 63124635
-43408529 17261147
-40575541 31131600
50801947 -107578603
53346243 -79817573
56568740 -75808437
65399272 -70026397
-44647800 14148329
-81907879 15250764
48793032 -92406773
84027853 -71557972
-66981999 -531321
-73320250 2891419
49201922 -89910746
-41515632 24167710
59198767 -73470661
-41597683 46953556
91251242 -77728502
87446178 -118346744
83029068 -71028007
97494680 -97939165
-73961262 67280172
63420011 -70865324
-41344211 45656485
-77697328 7595853
-65628263 71568937
95795690 -85912267
-78391089 62132640
-63094341 71807792
48589438 -94982801
-76006615 5492070
-61611534 -1015393
95670737 -106666640
-85401725 40872934
65777976 -69890344
95750168 -85785584
85635779 -119687755
64686565 -70303333
-80243845 59028334
97150185 -101052296
91501195 -78044461
-40534380 31744418
-77121678 6832893
85166962 -72240783
53939381 -78951705
-71737461 69007399
74902749 -123509011
-69923713 639172
61919076 -120580880
76432523 -123325069
-59788884 -709383
-79648335 10611508
90721733 -77090116
51882173 -109915504
-51727658 66920146
48883101 -100506345
-61785142 -1032045
95322707 -84675261
-85434216 30200299
-56876707 291812
53492024 -79598069
-57044202 70503029
79344104 -122666851
96359050 -104574715
-51040359 4465593
51130707 -108349937
-43540178 16895272
-48130914 7952870
-82439702 54109039
81926649 -70512602
-85069871 43644732
93590969 -81145374
82645481 -121394793
91372532 -77880600
-78161511 62469697
-45310526 12730372
48740899 -92864324
-46410097 10657557
96933974 -102233093
52436820 -81300615
-40767070 41694364
68230135 -69187581
-67331303 71146702
-54424827 1660391
79250710 -122694448
76116250 -123372060
49352966 -103014407
-84198644 48256532
96630896 -88668541
-41926999 48429091
85130941 -72217847
82206549 -70636789
50950239 -84301223
62023728 -120640479
73473970 -123583054
70671513 -123456211
-81340006 13931572
49173288 -90050875
-48522524 7412248
79854040 -69727081
-81739619 55872712
97500096 -97845337
91170841 -114606487
51315568 -108758285
