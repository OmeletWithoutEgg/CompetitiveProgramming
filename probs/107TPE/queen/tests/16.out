-11439090.36 43809063.99
